neurovault/apps/statmaps/views.py

```diff
@@ -550,6 +556,68 @@ def atlas_query_voxel(request):
         return JSONResponse('error: one or more coordinates are out of range', status=400)
     return JSONResponse(data)
 
+# Compare Two Images
+def compare_images(request,pk1,pk2):
+    image1 = get_image(pk1,None,request)
+    image2 = get_image(pk2,None,request)
+    
+    # Name the data file based on the new volumes
+    path, name1, ext = split_filename(image1.file.url)
+    path, name2, ext = split_filename(image2.file.url)
+
+    # image1.file.url is not correct --> /media/images does not exist
+    # hardcoded for now- for Neurovault can change volume1 to image1.file.url and it should work
+    volume1 = image1.file.url.replace("/media/images","/opt/image_data/images")
+    volume2 = image2.file.url.replace("/media/images","/opt/image_data/images")
```

Comment:
you need `image.file.path`, `image.file.url` is a virtual url used to obfuscate the real path